app/assets/stylesheets/mtl/extend/_buttons.scss

```diff
@@ -0,0 +1,8 @@
+.btn-icon {
```

Comment:
is this actually required, because there is afaik the option already to add icons to buttons as follows:

```html
<a href="#" class="btn">
  <i class="material-icons">add</i>
</a>
```